keyboards/sentraq/s60_x/readme.md

```diff
@@ -1,12 +1,12 @@
-S60-x
-=====
+# S60-x
```

Comment:
Suggestions for this file: https://github.com/qmk/qmk_firmware/commit/1446a2f2fbfb117aceb52842a6671fec8ac87418  
The edit to Line 13 is required, however. (Refactor causes `sentraq/s60_x` to no longer be a valid target.)

Either that, or fork this into two readmes, one for each version.